server/config/routes.js

```diff
@@ -378,20 +379,27 @@ exports.configure = function (app) {
   app.put('/prices/:uuid', priceList.update);
   app.delete('/prices/:uuid', priceList.delete);
 
-  // cash (aux/primary)
+  /** cash (aux/primary) */
   app.get('/cash', cash.list);
   app.get('/cash/:uuid', cash.getCashDetails);
   app.post('/cash', cash.create);
   app.put('/cash/:uuid', cash.update);
   app.delete('/cash/:uuid', cash.debitNote);
 
-  // @todo - classify these
+  /** @todo - classify these */
   app.get('/cashflow/report/', cashflow.getReport);
   //app.get('/stock/entries?', depot.getStockEntry);
 
-  // employees api
+  /** employees */
   app.get('/employees', employees.list);
   app.get('/employees/:id', employees.details);
   app.put('/employees/:id', employees.update);
   app.post('/employees', employees.create);
+
+  /** billing services */
+  app.get('/billing_services', billingServices.list);
+  app.get('/billing_services/:id', billingServices.details);
```

Comment:
The verb is in the HTTP request! 

In this case the verb might be _to detail_ vs _details_. If we were going to strictly follow the _create_, _update_, _list_ standards the verb here would be ```employees.detail```. Whatever we decide we should use uniformly everywhere.